plugins/extraction/big.ml

```diff
@@ -11,79 +11,77 @@
 (** [Big] : a wrapper around ocaml [Big_int] with nicer names,
     and a few extraction-specific constructions *)
 
-(** To be linked with [nums.(cma|cmxa)] *)
+(** To be linked with [zarith] *)
 
-open Big_int
-
-type big_int = Big_int.big_int
+type big_int = Z.t
```

Comment:
Have you tested this module? OCaml `Z` conflicts with Coq `Z` per https://github.com/coq/coq/pull/8252#discussion_r219342270. Given that PR we don't quite need to fix `Big` here.